create-customer/src/main/java/no/unit/nva/customer/create/CreateCustomerHandler.java

```diff
@@ -38,19 +55,25 @@ public CreateCustomerHandler() {
      * @param customerService customerService
      * @param environment   environment
      */
-    public CreateCustomerHandler(CustomerService customerService, Environment environment) {
-        super(Customer.class, environment, logger);
+    public CreateCustomerHandler(
+        CustomerService customerService,
+        CustomerMapper customerMapper,
+        Environment environment) {
+        super(CustomerDto.class, environment, logger);
         this.customerService = customerService;
+        this.customerMapper = customerMapper;
     }
 
     @Override
-    protected Customer processInput(Customer input, RequestInfo requestInfo, Context context)
+    protected CustomerDto processInput(CustomerDto input, RequestInfo requestInfo, Context context)
```

Comment:
So we should have a discussion about API objects, for another PR made by @axthosarouris he would simply call it `Customer` but it was namespaced under `api.model.{requests,responses}`, another alternative is prefix with `Api` on objects , or we ensure `CustomerDto` lives under some packagename with `api`

Reply:
It is maybe time we had another meeting in the team about common naming and structure.